code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -106,10 +106,10 @@
 	name = "Fireball"
 	spell_type = /obj/effect/proc_holder/spell/aimed/fireball
 
+/*
 /datum/spellbook_entry/rod_form
 	name = "Rod Form"
-	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form
-	cost = 40
+	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form */
```

Comment:
From what I can tell with this, you're cutting it out entirely now. That's even less logical. The rechargetime tweak you made further down already worked. No need to remove a feature.